clash-cores/src/Clash/Cores/Internal/UART.hs

```diff
@@ -0,0 +1,317 @@
+{-|
+  Copyright   :  (C) 2021, LUMI GUIDE FIETSDETECTIE B.V.
+  License     :  BSD2 (see the file LICENSE)
+  Maintainer  :  Rowan Goemans <goemansrowan@gmail.com>
```

Comment:
Do you want to keep the responsibility of maintainership? We can do that as well!

Reply:
I have set `QBayLogic B.V. <devops@qbaylogic.com>` to the maintainer.